src/fread.c

```diff
@@ -203,23 +203,23 @@ static void printTypes(int ncol) {
 }
 
 
-static inline void skip_white(const char **this) {
+static inline void skip_white(const char **ptr) {
```

Comment:
`this` wasn't a good name and `ptr` seems too generic too.   How about `myPos`, `threadPos`, `threadLoc`, `threadCh`, `myCh`, or something like that.   Some name to convey it's used to update the calling thread's position.

Reply:
Agree, `ptr` isn't very descriptive. However I'd avoid naming the parameter with anything "thread"-related -- this would create an impression that the functions have to be called from multi-threaded contexts only.
How about `inputPtr`, `inputCh`, `inputLoc` or `chPtr` ?